src/main/proto/user_service.proto

```diff
@@ -58,11 +59,36 @@ message CoopResponse {
     string logo = 4;
 }
 
+message PlatformManagerRequest {
+    string uuid = 1;
+    string coop = 2;
+}
+
+message UserWithExtendedInfoResponse {
```

Comment:
```suggestion
message UserWithInfoResponse {
```

Reply:
There is already a message with name UserWithInfoResponse, this would be duplicate then.

Reply:
Ok, my bad, then rename to `UserExtendedResponse`.